src/core_plugins/kbn_vislib_vis_types/public/pie.js

```diff
@@ -22,14 +22,14 @@ import { Schemas } from 'ui/vis/editors/default/schemas';
 import { CATEGORY } from 'ui/vis/vis_category';
 import pieTemplate from './editors/pie.html';
 
-export default function HistogramVisType(Private) {
+export default function HistogramVisType(Private, i18n) {
   const VisFactory = Private(VisFactoryProvider);
 
   return VisFactory.createVislibVisualization({
     name: 'pie',
```

Comment:
Please check maybe we also need to translate 'name'.

Reply:
As I can see we don't need translate `name`